.github/workflows/ci_cd.yml

```diff
@@ -120,6 +126,12 @@ jobs:
             python-version: '3.12'
 
     steps:
+      - uses: extractions/netrc@v2
+        with:
+          machine: github.com
+          username: pyansys-ci-bot
+          password: ${{ secrets.PYANSYS_CI_BOT_PACKAGE_TOKEN }}
+
       - name: Build wheelhouse and perform smoke test
         uses: ansys/actions/build-wheelhouse@v7
```

Comment:
This action allows you to skip the license check if you want to - but please add a TODO to solve this eventually: https://actions.docs.ansys.com/version/stable/build-actions/index.html#build-wheelhouse-action